src/traits/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod gomoku_ai;
+
+pub use gomoku_ai::GomokuAI;
```

Comment:
I don't really understand the use of this file, could you enlighten me please ?

Reply:
When using subfolders in cargo projects, we must in each subfolder: either define a .rs file with the same name as the subfolder, or define a mod.rs file in this subfolder. A subfolder define a module, and this file describes what will be publicly exposed by the module.

I needed to add it for `rust-analyzer` to work, since it is tied to cargo, and thus can only analise code that it part of our project, so we need to define that is used for it be part of the cargo project

Reply:
Okay thank you 🫡